### ISSUE TITLE
HybridWebView should derive from WebView and HybridWebViewHandler from WebViewHandler

### ISSUE THREAD

**Issue by Shishim4ru** [nmoschkin/MAUIWebViewExample#4]
Changing the base class of the HybridWebView and of the handler allows to utilize the properties, methods and events of the WebView, such as Eval, EvaluateJavascriptAsync or Navigated.

In my fork I have already made these changes and it seems to work well (tested only on Windows)

**Comment by nmoschkin** [comment]
Yes. I agree. But, for some people it was not working as expected, so I tried it this way, and it did work. Feel free to modify it. I suspect a final solution from Microsoft will come along as MAUI gets more refined.
